metadata.json

```diff
@@ -30,7 +30,7 @@
     },
     {
       "name": "katello/qpid",
-      "version_requirement": ">= 7.0.0 < 8.0.0"
+      "version_requirement": ">= 7.1.0 < 8.0.0"
```

Comment:
7.1.0 doesn't contain the patches needed. Making a note here so we don't forget to update it in the final version.